src/models/github-server-app.ts

```diff
@@ -97,14 +97,10 @@ export class GitHubServerApp extends EncryptedModel {
 	 * @param installationId
 	 * @returns {GitHubServerApp[]}
 	 */
-	static async getAllForGitHubBaseUrl(
```

Comment:
Renamed for clarity and modified so we're returning an empty array if no entries are found (same as Subscription and Installation).